### DISCUSSION TITLE
How to submit secondary header search on user press enter version 3

### DISCUSSION THREAD

**Discussion by fabiomlferreira** [rappasoft/laravel-livewire-tables#1396] [Q&A]
I'm implementing for the first time laravel livewire tables, it's a very complete extension but it's too complicated with too many options.
I'm trying to do simple things, at the moment I have a secondary header with a text filter but it only submit on blur, I think this is not user friendly should be debounce or on enter key press.

Is possible to set that?

**Comment by lrljoe** [comment]
Can you open a feature request for this in the issues tab please.

It won't take too long to do fortunately, definitely this month, just writing the docs and tests will take a bit of time!

**Comment by lrljoe** [comment]
Closed as feature request exists
